all-spark-notebook/Dockerfile

```diff
@@ -98,9 +80,10 @@ RUN conda install --quiet --yes \
     'r-ggplot2=1.0*' \
     'r-rcurl=1.95*' && conda clean -tipsy
 
-# Scala Spark kernel spec
-RUN mkdir -p /opt/conda/share/jupyter/kernels/scala
-COPY kernel.json /opt/conda/share/jupyter/kernels/scala/
+# Toree kernel
+ENV TOREE_VERSION ~=0.1.0.dev
+RUN pip install toree$TOREE_VERSION
```

Comment:
Oh man... doing too many of these at the same time. It totally does not need to be an ENV. :) I'll inline that.